mango-viz/bdgenomics/mango/io/VCFfile.py

```diff
@@ -0,0 +1,128 @@
+#
+# Licensed to Big Data Genomics (BDG) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional insourceion
+# regarding copyright ownership.  The BDG licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+from .genomicfile import GenomicFile
+from collections import OrderedDict
+import gzip
+
+class VCFFile(GenomicFile):
+
+    global VCF_HEADER
+    global NUM_SAMPLES
+    VCF_HEADER = ['CHROM', 'POS', 'ID', 'REF', 'ALT', 'QUAL', 'FILTER', 'INFO', 'FORMAT']
+
+
+    @classmethod
+    def _read(cls, filename, large = True):
+        """Open an optionally gzipped VCF file and return a pandas.DataFrame with
+        each INFO field included as a column in the dataframe.
+        
+        Borrowed from https://gist.github.com/slowkow/6215557
+
+        Note: Using large=False with large VCF files. It will be painfully slow.
+        :param filename:    An optionally gzipped VCF file.
```

Comment:
I am a little confused about the purpose of the large parameter. It looks like if large=False, nothing happens. Is this correct?